_maps/configs/syndicate_gorlex_phantom.json

```diff
@@ -0,0 +1,42 @@
+{
+	"faction": "/datum/faction/syndicate",
+	"prefix": "ISV",
+	"namelists": [
+		"GORLEX",
+		"BEASTS",
+		"WEAPONS"
+	],
+	"map_name": "Phantom-class Mining Courier",
+	"map_short_name": "Phantom-class",
+	"description": "A modern ship only made in the past twenty-ish years, the Phantom's sleek hull, narrow profile, and wings would make one to believe it a racer, or small yacht. Inside, however, is a sturdy layout for small-scale Hardliner mining groups, commissioned by Cybersun and built by ISF.",
+	"tags": [
+		"Mining"
+	],
+	"map_path": "_maps/shuttles/syndicate/syndicate_gorlex_phantom.dmm",
```

Comment:
Hey, rename it to `syndicate_hardliner_phantom.dmm`. The "gorlex" part is outdated, and is being slowly removed on the NGR side of things.